AnagramValidityChecker.java

```diff
@@ -0,0 +1,83 @@
+package CodeUAss1Package;
+
+import java.util.HashMap;
+
+public class AnagramValidityChecker
+{
+    /* Given two strings, determine if one is an anagram of the other.

```

Comment:
Please follow javadoc conventions in class and method comments. Typical format is:
```
/**
 * Sentence with summary of what method does.
 *
 * <p>More sentences that clarify logic.
 */